internal/yunikorn/sync.go

```diff
@@ -131,10 +131,10 @@ func (s *Service) upsertPartitionQueues(ctx context.Context, partitions []*dao.P
 		return nil, fmt.Errorf("failed to get queues for some partitions: %v", errs)
 	}
 
-	queues = flattenQueues(queues)
+	// queues = flattenQueues(queues)
```

Comment:
Remove this line?

Reply:
Also probably we can remove the `flattenQueues()` ?

Reply:
Aha - just realized the only remaining real reference to `flattenQueues()` was its unit-test! Now removed in commit b6628d4.  Good catch.